mtp_api/apps/prison/models.py

```diff
@@ -23,9 +23,12 @@ class PrisonerLocation(TimeStampedModel):
     prison = models.ForeignKey(Prison)
 
     class Meta:
-        index_together = [
-            ["prisoner_number", "prisoner_dob"],
-        ]
+        index_together = (
+            ('prisoner_number', 'prisoner_dob'),
+        )
+        permissions = (
+            ('identify_prisoner', 'Can identify if a prisoner exists'),
```

Comment:
I can't see that you define what this permission means anywhere. In other places this is done by creating an ActionsBasedPermissions subclass which links the permission to a view method.